test/test_pynisher.py

```diff
@@ -201,7 +201,10 @@ def test_out_of_memory(self):
         for mem in [1024, 2048, 4096]:
             self.assertIsNone(wrapped_function(mem, 0, 0))
             self.assertEqual(wrapped_function.exit_status, pynisher.MemorylimitException)
-            self.assertEqual(wrapped_function.exitcode, 0)
+            # In github actions, randomly on python 3.6 and 3.9, the exit
+            # status is 1 that happens while running ppid_map during a MemoryError
+            # This happens randomly -- sometimes in conda others in the env python version
+            self.assertIn(wrapped_function.exitcode, (1, 0))
```

Comment:
Do you know if this is restricted to certain contexts, too?

Reply:
I have seen this happen in:

- 3.7 python forkserver a seen in [here](https://github.com/automl/pynisher/runs/1566065742)

- 3.6 python fork as seen in [here](https://github.com/automl/pynisher/runs/1566065442)

Because of this, I did not encapsulate this under an if context equal something flag.

Reply:
Alright, let's then merge this one.